server/api/config.go

```diff
@@ -145,6 +145,9 @@ func (h *confHandler) updateConfig(cfg *config.Config, key string, value interfa
 	kp := strings.Split(key, ".")
 	switch kp[0] {
 	case "schedule":
+		if h.svr.IsTTLConfigExist(key) {
+			return errors.Errorf("need to clean up TTL first for %s", key)
```

Comment:
How about 
```suggestion
			return errors.Errorf("Firstly,  you need to clean up the TTL for %s", key)
```

Reply:
IMO, there is not much difference since this function may also be called by other components, the `you` here is not important.